.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "endOfLine": "auto",
```

Comment:
Please keep the prettier rules the same.

Reply:
Removed the line